CHANGES

```diff
@@ -0,0 +1,92 @@
+0.2.2 Tue Mar 12 22:05:30 2019 +0100
```

Comment:
I might drop the timezones from the dates.

Reply:
done